torch/ao/quantization/quantize_fx.py

```diff
@@ -340,6 +344,7 @@ def fuse_fx(
 def prepare_fx(
     model: torch.nn.Module,
     qconfig_dict: Any,
+    example_inputs: ExampleInputs,
```

Comment:
since this is a user facing API, it would be good to consider spelling out the type explicitly so the user does not have to navigate to a different file to find it, something like

```
example_inputs: Tuple[Union[torch.Tensor, Any], ...]
```

Reply:
yeah, but we can have all types of inputs including (Tensor, List, Dict, Tuple etc. and nested structures with these things)